/bin/sh is not guranteed to handle bashisms. Switch interpreter of
safe_asterisk to /bin/bash.

 -- Mark Purcell <msp@debian.org>

--- asterisk-1.4.9~dfsg.orig/contrib/scripts/safe_asterisk
+++ asterisk-1.4.9~dfsg/contrib/scripts/safe_asterisk
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 # vim:textwidth=80:tabstop=4:shiftwidth=4:smartindent:autoindent
 
 CLIARGS="$*"				# Grab any args passed to safe_asterisk
